main.js

```diff
@@ -0,0 +1,13 @@
+const toggleIcon = document.getElementById('toggleIcon');
+const toggleElement = document.getElementById('dropdownClick');
```

Comment:
- [ ] When I switch to the mobile version the menu is dropdown.

![jjjj](https://user-images.githubusercontent.com/61562626/140936738-8c3ac8be-c140-4080-8269-2cbd5b59dd86.gif)

Reply:
@dasileker  Thank you for the review and suggestion. It has taken a lot of time to figure out but finally, I did it 😃